backend/docs/development_docs/api_requests_per_screen.rst

```diff
@@ -70,54 +76,109 @@ SOLR will return data in JSON format and according to the following template: ::
      (...)
      },
     "response":{"numFound":1,"start":0,"docs":[
-      {
-        "uuid":["97337de5-6096-41f9-921e-5bed7a140d85"],
-        "name":["Hjørring"],
-        "parent":["ROOT"],
-        "locations":["DAR",
-          "Stensgårdsvej 19, Nr Harritsl, 9800 Hjørring"],
-        "employees":["Esther Hildeborg Nielsen",
-          "2355a754-5de4-4b3f-b78c-0dceba80dfa6",
-          "Ergoterapeut",
-          (...)
-          "Pawel Nyskov Dalgaard",
-          "edb1f226-90db-4d23-94d2-232c048f84de",
-          "Jurist"],
-        "departments":["Borgmesterens Afdeling",
-          "40644200-b3f1-42d4-8752-8dab581d5b23",
-          (...)
-          "Teknik og Miljø",
-          "be7bb9e5-b298-4019-b0fd-ce80c8f97934"],
-        "associated":[
-          "Christian Kjeldal Schmidt Nielsen",
-          "4419dbb3-a698-4d98-b255-a4eff621ee4f",
-          "Problemknuser",
-          "Specialkonsulent"],
-        "managers":["Direktør",
-          "Esther Hildeborg Nielsen",
-          "b078107a-4299-4251-b41c-23e0b8a9575a"],
-        "id":"38bbb6ad-d31f-4381-aa88-61be37f70d8c",
-        "_version_":1616659562091249664}]
+        {
+           "document":"<very long string omitted>",
+        "id":"8284b4b5-118a-4cbf-aa7c-f08ae2ca391c",
```

Comment:
Indentation is weird. Also, should probably be styled as JSON, assuming you edit it to be valid.

Reply:
Will indent & mark as JSON.